src/cookies.js

```diff
@@ -198,16 +210,38 @@ function trySetCookie(win, name, value, expirationTime, domain) {
   }
 }
 
+/**
+ * Gets the cookie string to use for SameSite. This works around an issue where
+ * Safari treats an unknown value ("None") as "Strict".
+ * @param {Window} win
+ * @param {!SameSite|undefined} sameSite
+ * @return {string} The string to use when setting the cookie.
+ */
+function getSameSiteString(win, sameSite) {
+  if (!sameSite) {
+    return '';
+  }
+
+  const platform = Services.platformFor(win);
+  const isWebkit = platform.isIos() || platform.isSafari();
```

Comment:
Is Chrome's behavior on iOS also broken?

Reply:
Yes, but after researching this a bit more,it looks like the issue is fixed in iOS 13, which was just recently released. For the experiments page, I think it would be fine to just omit the platform check and have this cookie not work correctly on iOS 12. I'm less familiar with the other uses of `setCookie` in cid-impl and amp-analytics. For those usages, I assume we would want to check for iOS at least until 14 is released.

Reply:
Actually, on second thought, I'm not sure samesite even matters for those cookies (they are set on the domain itself, not for a subresource), so it is probably okay to just remove the platform checks.